server-phoenix/mix.exs

```diff
@@ -63,7 +64,8 @@ defmodule Helios.MixProject do
       {:absinthe_phoenix, "~> 2.0.0"},
       {:uuid, "~> 1.1"},
       {:exq, "~> 0.16.1"},
-      {:quantum, "~> 3.0"}
+      {:quantum, "~> 3.0"},
+      {:kaffy, git: "https://github.com/malssid/kaffy.git"}
```

Comment:
Was it ever discussed to dogfood our own [torch](https://github.com/mojotech/torch) package instead of using kaffy? Or is this more of an experiment?

Reply:
😞